### DISCUSSION TITLE
Installation

### DISCUSSION THREAD

**Discussion by ThibaultMatthieuSNCF** [labs4capella/python4capella#143] [General]
Hello,

I have followed the installation instruction but I'm not able to run python script.
In the menu when I right click in the script I don't have the "Run as.." field.

my menu.png : What I get 

![my menu](https://user-images.githubusercontent.com/54263766/186208681-f5457c4d-b822-4174-9d63-953fc5bb8456.png)

normal menu : What I should have
![Normal menu](https://user-images.githubusercontent.com/54263766/186207889-334d48c8-c231-40e4-a69f-4d9978dfedd3.png)

OS : Windows 10
Capella version : 5.1.0 & 5.2.0

Thanks for your help

**Comment by Sophie-Pl** [comment]
Hello,
Did you enable Developer capabilities ( Window > Preferences > General > Capabilities > Development) ?

 Best regards,

    Sophie

**Reply by ThibaultMatthieuSNCF** [reply]
Hello,
No I don't
Now it works !

Thank you

Matthieu

**Comment by guc17** [comment]
<img width="483" height="292" alt="image" src="https://github.com/user-attachments/assets/dc0218ad-07d4-4fd0-b206-9a2176924d65" />

<img width="1405" height="267" alt="image" src="https://github.com/user-attachments/assets/4efedee7-b8f5-486b-997f-bb6cbbeb5de3" />

I get this error when trying to run the sample_scripts. I clicked on Run As -> EASE script and this is the error I get. Any suggestions are appreciated !

**Comment by ylussaud** [comment]
I think you missed one step in the [configuration](https://github.com/labs4capella/python4capella?tab=readme-ov-file#python-for-capella-configuration). You need to define a Python interpreter:

- select the Scripting / Python Scripting (using Py4J)
- select the python executable from your system

**Reply by guc17** [reply]
Hello, I did select my python interpreter but I forgot this step 'python -m pip install openpyxl'. Now I have progressed but hit a new road block: 
C:\Users\uday.chandra\Downloads\capella-7.0.1.202503211540-win32-win32-x86_64\capella\plugins\org.eclipse.ease.lang.python.py4j_0.8.0.I202104091518\pysrc\ease_py4j_main.py:414: DeprecationWarning: setDaemon() is deprecated, set the daemon attribute instead
  thread.setDaemon(True)

**Comment by ylussaud** [comment]
This is a warning but if you plan to use numpy you probably want to comment those lines. See this [issue](https://github.com/labs4capella/python4capella/issues/112) for more details.
If you prefer you can replace the call to the setter with a direct access to the attribute:
```
thread.daemon = True
```
